.travis.yml

```diff
@@ -23,3 +23,9 @@ script:
   - make build phaul test/test test/phaul test/piggie
   # Run actual test as root as it uses CRIU.
   - sudo make test phaul-test
+  # This builds crit-go
+  - cd crit-go
+  - cd magic-gen
+  - make build magicgen
+  # Run test of crit-go
+  - sudo make test 
```

Comment:
```
$ cd crit-go
/home/travis/.travis/functions: line 109: cd: crit-go: No such file or directory
The command "cd crit-go" exited with 1.
$ cd magic-gen
/home/travis/.travis/functions: line 109: cd: magic-gen: No such file or directory
The command "cd magic-gen" exited with 1.
$ make build magicgen
make: *** No rule to make target 'magicgen'.  Stop.
The command "make build magicgen" exited with 2.
```
Hi @Pac23, you need to push this commit in your `magic_gen` branch (https://github.com/checkpoint-restore/go-criu/pull/21) because it depends on the changes introduced there.

```
cd crit-go
cd magic-gen
make build magicgen
sudo make test 
```
The above four commands are equivalent to `sudo make -C crit-go/magic-gen/`

Reply:
i have made the changes,pushed travis.yml to the other branch.